templates/partials/summary/collapsible-summary.html

```diff
@@ -7,7 +7,7 @@
   {%- if group["blocks"] | length -%}
     {%- if group.title -%}
       {%- set item = {
-        "title": group.title,
+        "groupTitle": group.title,
```

Comment:
This doesn't work for me. When I change `groupTitle` to `title` then it is displayed on the summary.

Reply:
I think we should also update one of our tests to ensure that the group title is displayed as expected, as this doesn't seem to be tested currently.